templates/revoked_keys.j2

```diff
@@ -0,0 +1,3 @@
+{% for key in ssh_server_revoked_keys %}
```

Comment:
Added. I also checked and the comment doesn't interfere with the functionality. :+1: